### DISCUSSION TITLE
Show saved CO2

### DISCUSSION THREAD

**Discussion by schiessle** [Traewelling/traewelling#2327] [Ideas / Feature Requests]
Hi all,

I just saw it in the "Bahn Bonus" app of DB in Germany. They show the user how much CO2 they saved with their train ride compared to a car and how long they could heat their house with it.

I know it is not easy to calculate and you have to make quite some assumptions. But i think it is a nice gamification even if it will never be 100% accurate. Could be also nice for Traewelling.

**Comment by ChillLP** [comment]
Looking at the fact that you can Check-in into buses, DMUs and so on i dont think you can "save" a Lot of co2

**Reply by juengerjustus** [reply]
Yes, of course you're right in this case. But especially in trains that are travelled over a long distance, I think you can notice a considerable difference.

**Comment by juengerjustus** [comment]
Very interesting for the user to see.
But certainly difficult to implement. On the one hand, capacity utilisation plays a role, and therefore also the capacity of the vehicle. Does Träwelling know through the API which vehicles/series are running? Also from private companies? Without this information, it is probably only possible to make very generalised and vague calculations.
You can certainly work with flat rates for buses, but what about ferries, for example?

**Comment by marhei** [comment]
Sounds more like greenwashing, claiming CO2 savings for rather unnecessary journeys that the bubble would never have made anyway, other than by train.
